dialog/file_test.go

```diff
@@ -341,9 +352,10 @@ func TestShowFileSave(t *testing.T) {
 		return
 	}
 
-	// This will only execute if we have a file in the home path.
-	// Until we have a way to set the directory of an open file dialog.
-	test.Tap(target)
+	abs, err := filepath.Abs("./testdata/")
```

Comment:
From static analysis: `his value of err is never used`

Reply:
Thanks, I missed that as I was iterating the test